packages/pubsub/src/index.js

```diff
@@ -722,22 +681,10 @@ PubSub.prototype.snapshot = function(name) {
  * requests. You will receive a {module:pubsub/subscription} object,
  * which will allow you to interact with a subscription.
  *
- * All generated names share a common prefix, `autogenerated-`.
+ * @throws {Error} If subscription name is omitted.
  *
- * @param {string=} name - The name of the subscription. If a name is not
- *     provided, a random subscription name will be generated.
+ * @param {string} name - Name of the subscription.
  * @param {object=} options - Configuration object.
```

Comment:
We need to show the options here for `flowControl` (and whatever else is relevant).